otx/algorithms/detection/adapters/mmdet/utils/config_utils.py

```diff
@@ -186,7 +185,6 @@ def prepare_for_training(
     learning_curves: defaultdict,
 ) -> Config:
     """Prepare configs for training phase."""
-    config = copy.deepcopy(config)
```

Comment:
Let's do this by judging whether there is an abnormality through a test.

Reply:
@harimkang I think that Pre-Merge-Test already shows this is okay. What do you think?
If It is not enough, I can add mmaction's prepare_for_training fucntion

Reply:
If the pre-merge test has passed, let's put it like this.